plugin/src/test/java/org/opensearch/ml/rest/RestCohereInferenceIT.java

```diff
@@ -0,0 +1,76 @@
+package org.opensearch.ml.rest;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Before;
+import org.opensearch.ml.common.FunctionName;
+import org.opensearch.ml.common.dataset.TextDocsInputDataSet;
+import org.opensearch.ml.common.input.MLInput;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public class RestCohereInferenceIT extends MLCommonsRestTestCase {
```

Comment:
This IT can be merged into the RemoteInferenceIT?

Reply:
The RemoteInferenceIT class file is already too big, I prefer we use separate file to test single feature for better maintenance.

Reply:
btw, the spotless is not applied yet.